test/powershell/Language/Classes/Scripting.Classes.Attributes.Tests.ps1

```diff
@@ -401,7 +401,7 @@ Describe 'ValidateSet support a dynamically generated set' -Tag "CI" {
                 Import-Module -Name $moduleFile -Force
                 Test-ValidateSet 'Hello' | Should -BeExactly 'Hello'
             } finally {
-                Remove-Module -Name $moduleFile -Force
+                Remove-Module -Name $moduleFile -Force -ErrorAction SilentlyContinue
```

Comment:
This is not related to this formatting code change, right?

Reply:
Correct.  Was running the tests locally and this line was outputting unncessarily